.github/workflows/publish-release.yml

```
@@ -0,0 +1,91 @@
+name: publish-release
+
+on:
+  release:
+    types: [published]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx2g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+  create_staging_repository:
+    runs-on: ubuntu-latest
+    name: Create staging repository
+    outputs:
+      repository_id: ${{ steps.create.outputs.repository_id }}
+    steps:
+      - id: create
+        uses: nexus-actions/create-nexus-staging-repo@v1.1
```

You might want to replace the version with its sha1 to make sure the tag isn't moved under the hood:

```suggestion
        uses: nexus-actions/create-nexus-staging-repo@10a43a22d0278ed006f1293214b99184c60fbdff
```

I am a maintainer of this action and I can assure you there is nothing fishy going on down there but you shouldn't trust me :)

---

Nice, thanks! Not super familiar with Github actions versioning.